setup.cfg

```diff
@@ -37,6 +38,7 @@ license_files =
     licenses/LICENSE_textmate_toml.tmbundle
     licenses/LICENSE_vscode-restructuredtext_vscode-restructuredtext
 classifiers =
+    License :: OSI Approved :: Eclipse Public License 2.0 (EPL-2.0)
```

Comment:
hmmm this shouldn't have changed, unless EPL requires relicensing and then we probably can't use this repo :(